devops/pollers/.env.home.example

```
@@ -1,14 +1,22 @@
-# For Fly.io (remote-poller via Tailscale): http://192.168.1.81:8080
-TURSO_DATABASE_URL=http://staktrakr-sqld:8080
-TURSO_AUTH_TOKEN=your-token
-
-# Nightly DR sync to Turso Cloud free tier (turso-backup-sync.js)
+# Primary database — choose ONE of the two options below.
+#
+# Option A (default, what lbruton/StakTrakr runs): local self-hosted sqld
+#   - Docker (home-poller):       http://staktrakr-sqld:8080
+#   - Fly.io (via Tailscale):     http://192.168.1.81:8080
 SQLD_URL=http://staktrakr-sqld:8080
+SQLD_AUTH_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SQLD_AUTH_TOKEN key should go before the SQLD_URL key

(UnorderedKey)